docs/docs/api/Tasks/fetchSingleTask.md

```diff
@@ -22,53 +22,36 @@ This is a **protected route**, a **valid JWT is required** in the header field
 ```
 Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
-
-#### Body
-
-```json
-{}
-```
-
 #### Parameters
 
-    id: task_id // Id of the task that is being searched for
+  id: task_id 
 
 ### Possible Responses
 
 #### Immediate Success
 
 ```json
 {
-  "code": 200,
-  "message": "",
   "data": {
     // Task information
   }
 }
 ```
 
-#### Failure
+#### Immediate Failure
```

Comment:
Minor comment: Just for consistency, this should just be Failure rather than Immediate Failure (since the other files just note Failure)